plugins/opsgenie/icon_opsgenie/util/api.py

```diff
@@ -40,6 +52,32 @@ def test_api(self) -> dict:
         GET_TEST_URL = f"{self.api_url}alerts/count"
         return self._call_api("GET", GET_TEST_URL)
 
+    def _rate_limiting(max_tries: int) -> dict:
```

Comment:
Does a linear backoff actually work with their API to prevent further rate limiting? I'm not sure, but it may be a good idea to use an exponential backoff instead.

Reply:
Ok. Updated :smiley: